app/src/main/java/com/stevesoltys/seedvault/restore/RestoreViewModel.kt

```diff
@@ -294,8 +290,12 @@ internal class RestoreViewModel(
     }
 
     @WorkerThread
-    private inner class RestoreObserver(private val startRestore: (RestoreObserver) -> Unit) :
-        IRestoreObserver.Stub() {
+    private inner class RestoreObserver(
+        private val session: IRestoreSession,
+        private val token: Long,
+        private val packages: MutableList<String>,
```

Comment:
Does the list need to be mutable here? (Good practice to build API where the recipient can't modify your objects if they don't need to)